dev/rm-config.yaml

```diff
@@ -9,4 +9,4 @@ db:
   dsn: "root:@tcp(127.0.0.1:3306)/spincycle_dev"
 jr_client:
   server_url: "http://127.0.0.1:9999"
-spec_file_dir: "../specs/"
+spec_file_dir: "specs/"
```

Comment:
Could you add a 5th assumption to the top of the README (but put it right after the jobs symlink assumption) that says something like "The specs symlink points to the specs you want (default: dev/specs)"